test/suites/completions.sh

```diff
@@ -169,10 +169,10 @@ test_completions() {
     [ "$(complete config get localhost:c1 '')" = 'boot.,cloud-init.,cluster.,environment.,limits.,linux.,migration.,nvidia.,placement.,raw.,security.,snapshots.,ubuntu_pro.,user.' ]
     [ "$(complete config get c1 limits.)" = 'limits.cpu,limits.cpu.,limits.disk.,limits.hugepages.,limits.kernel.,limits.memory,limits.memory.,limits.processes' ]
     lxc config set user.foo=bar
-    [ "$(complete config unset '')" = 'c1,c2,core.https_address,localhost:,user.foo' ]
+    [ "$(complete config unset '')" = 'c1,c2,core.https_address,localhost:,network.ovs.connection,user.foo' ]
     [ "$(complete config unset c)" = 'c1,c2,core.https_address' ]
     [ "$(complete config unset l)" = 'localhost:' ]
-    [ "$(complete config unset localhost: '')" = 'core.https_address,user.foo' ]
+    [ "$(complete config unset localhost: '')" = 'core.https_address,network.ovs.connection,user.foo' ]
```

Comment:
Won't this config key only be set if `LXD_OVS_CONNECTION` is set in the environment?